packages/blockchain-wallet-v4-frontend/src/modals/BuySell/CheckoutConfirm/template.success.tsx

```diff
@@ -201,17 +201,6 @@ const Success: React.FC<InjectedFormProps<{ form: string }, Props> & Props> = (p
     props.buySellActions.cancelOrder(props.order)
   }
 
-  const paymentPartnerButton =
-    paymentPartner === BankPartners.YAPILY ? (
-      <FormattedMessage id='copy.next' defaultMessage='Next' />
```

Comment:
Are you sure it's ok to remove this text? When you make a buy with yapily you don't go to the order confirmation screen after this screen. You go to an approval screen, and then on to the order confirmation screen.